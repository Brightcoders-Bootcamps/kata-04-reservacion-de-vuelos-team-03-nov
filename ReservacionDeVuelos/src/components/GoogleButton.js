import React from 'react';
import {Text, TouchableOpacity} from 'react-native';

const GoogleButton = () => {
  return (
    <TouchableOpacity>
      <Text>Hey, it works!</Text>
    </TouchableOpacity>
  );
};

export default GoogleButton;
