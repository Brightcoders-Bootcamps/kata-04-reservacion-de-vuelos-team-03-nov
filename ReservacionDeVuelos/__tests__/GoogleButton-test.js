import React from 'react';
import renderer from 'react-test-renderer';
import GoogleButton from '../src/components/GoogleButton';

test('renders correctly', () => {
  const tree = renderer.create(<GoogleButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
