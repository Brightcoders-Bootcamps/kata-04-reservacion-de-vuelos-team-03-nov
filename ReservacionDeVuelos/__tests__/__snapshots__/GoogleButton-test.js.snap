// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": undefined,
      "borderRadius": 10,
      "flexDirection": "row",
      "height": 88.93333333333334,
      "marginTop": 10,
      "opacity": 1,
      "padding": 10,
      "width": "85%",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "width": "20%",
      }
    }
  >
    <Image
      source={
        Object {
          "process": [Function],
        }
      }
      style={
        Object {
          "height": 30,
          "width": 30,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "fontSize": 16,
            "fontWeight": "bold",
          },
          Object {
            "color": undefined,
          },
        ]
      }
    />
  </View>
</View>
`;
